### PR TITLE
Rotary Encoder Switches / Sources removed

### PR DESCRIPTION
**Pull request by bsongis**

### CODE REVIEW

**Comment by dgatf** [comment]
SWSRC_REa and TR_ROTENC_SWITCHES has to be removed from the radio firmware

File: companion/src/modeledit/flightmodes.cpp [hunk 1]
```diff
@@ -40,7 +40,6 @@ FlightModePanel::FlightModePanel(QWidget * parent, ModelData & model, int phaseI
   board = getCurrentBoard();
   fmCount = firmware->getCapability(FlightModes);
   gvCount = firmware->getCapability(Gvars);
-  reCount = firmware->getCapability(RotaryEncoders);
```
Comment (dgatf) [inline companion/src/modeledit/flightmodes.cpp:43]: Also remove _for loops_  (x7) with reCount below or it crashes companion

**Review by dgatf — COMMENTED** [review]
Also it is needed to remove from firmware:
`STR_ROTENC_SWITCHES, SWSRC_REa`

Below some functions, widgets and vars that are unused in companion. Maybe you prefer to keep it though
```
rotaryEncoders
getREncValue
isREncLinked
isREncLinkedCircular
linkedREncFlightModeZero
reValues
reUse
linkedREncFlightModeZero
```

**Comment by bsongis** [comment]
May I ask you to continue on this? I am afraid I won't have any spare time during this week
By the way it will be part of 2.3.9 (not 2.3.8) as it will be a huge PR ...

**Comment by dgatf** [comment]
Sure, I continue with it

**Comment by bsongis** [comment]
@dgatf May I ask you to work on this in 2.4? I don't think it is a good idea to do it in 2.3 ...

**Comment by dgatf** [comment]
Sure, no problem

**Pull request closed without merging**